lib/Ohmbrewer_Menu/Ohmbrewer_Menu_Home.cpp

```diff
@@ -9,21 +9,44 @@
  * @param settings A pointer to the RuntimeSettings object.
  */
 Ohmbrewer::MenuHome::MenuHome(Ohmbrewer::Screen *screen, Ohmbrewer::RuntimeSettings *settings) : Ohmbrewer::Menu::Menu(screen, settings) {
-    // Nothing new to do
+    options.push_back("Temp Sensors");
```

Comment:
If we're switching things up to work this way, we should include an option for standalone pumps.